CONTRIBUTING.md

```diff
@@ -91,19 +91,19 @@ The following checklist should be followed when creating a release:
   - [ ] Name the release the same as the tag.
   - [ ] Include in the release notes all changes from CHANGELOG that are being released.
     - [ ] Attach the relevant assets to the release, generated previously by `./build`:
-      - [ ] Attach `dist/summon-darwin-amd64.tar.gz` to release.
-      - [ ] Attach `dist/summon-linux-amd64.tar.gz` to release.
-      - [ ] Attach `dist/summon-windows-amd64.tar.gz` to release.
-      - [ ] Attach `dist/summon_v*.rpm` to release.
-      - [ ] Attach `dist/summon_v*.deb` to release.
-      - [ ] Attach `dist/CHANGELOG.md` to release.
-      - [ ] Attach `dist/SHA256SUMS.txt` to the release.
+      - [ ] Attach `dist/goreleaser/summon-darwin-amd64.tar.gz` to release.
+      - [ ] Attach `dist/goreleaser/summon-linux-amd64.tar.gz` to release.
+      - [ ] Attach `dist/goreleaser/summon-windows-amd64.tar.gz` to release.
+      - [ ] Attach `dist/goreleaser/summon_v*.rpm` to release.
+      - [ ] Attach `dist/goreleaser/summon_v*.deb` to release.
+      - [ ] Attach `dist/goreleaser/CHANGELOG.md` to release.
+      - [ ] Attach `dist/goreleaser/SHA256SUMS.txt` to the release.
```

Comment:
Question: Why do we need a `goreleaser` subdirectory? Are there other ways we'll build the binaries in the future, or is this just the default output directory of goreleaser?

Reply:
Consistency more than anything. summon-conjur and another project are using the subdir